_drafts/advanced-power.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: wiki
+title: Opening / Changing Advanced Power Options
+description: "Opening / Changing Advanced Power Options using .bat files"
+parent: Tips and Tricks
+grand_parent: General
+contributors:  
+discord: ['ecto UK/ES#2667']
+---
+
+# Opening / Changing Advanced Power Options using .BAT files
+
+{. :highlight}
+This could be useful if you want to quickly change settings in your power plan based on what your laptop is doing.
+
+## Shortcut to open advanced settings
+
+Create a new text document and copy/paste the line from below Then save it
+
+```
+AdvPowerSettings.batcontrol.exe powercfg.cpl,,3
+```
+
+## How to change settings
+
+Create a new text document and copy/paste one of the lines from below and Save it as
+
+### Boost Disabled
+-------------------------------
```

Comment:
![image](https://user-images.githubusercontent.com/85416491/184555673-471741bc-1f03-4c30-beb0-bb2ea23a2f0f.png)

the different boost modes need to have a difference compared to headings, so that it is easier for user to understand the commands next to it, makes it even better when it is being viewed. looks wise, looks better imo

Reply:
changed it to usual ### without the underline